app/components/toggler_dropdown_component.rb

```diff
@@ -0,0 +1,17 @@
+# Put content in togglable box that is closed by default
+
+class TogglerDropdownComponent < ApplicationComponent
```

Comment:
Made this to 'deprecate' things in the admin interface, but I've decided not to use it. Have left it here for now.